package.json

```diff
@@ -45,6 +45,12 @@
   ],
   "devDependencies": {
     "@babel/cli": "^7.16.8",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-export-namespace-from": "^7.16.7",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.16.7",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.16.7",
+    "@babel/plugin-proposal-optional-chaining": "^7.16.7",
+    "@babel/plugin-transform-modules-commonjs": "^7.17.9",
```

Comment:
Why do we have some of these modules here? Is it to build our own source?

Reply:
Yes, our build has a babel step.